### DISCUSSION TITLE
add custom link to Whoogle

### DISCUSSION THREAD

**Discussion by ghostersk** [benbusby/whoogle-search#621] [Ideas]
I know the reason for using whoogle is to stop google to track you.
But it would be amazing if it is possible to add shortcuts to main page and search page for Gmail and google maps.
Especially for Gmail, so i can access my email quickly without  need of another bookmark, just by going to homepage of browser.
If anyone know where to edit the HTML or if there is option to add my shortcuts would be very much appriciated.
Thanks

**Comment by Albonycal** [comment]
all the static HTML files are in `app/templates`
also you can make your own custom HTML and make it send a `GET` request to the whoogle instance 
something like this
```html

<form method="get" action="https://search.albony.xyz/search">
   <input type="text" style="width: 50%;border: none;background-color: rgba(255, 255, 240, 0.7);" name="q" size="31" value=""  id="search" autocomplete="off" placeholder="Search">
   </form>
```
with some CSS
